py3status/modules/spotify.py

```diff
@@ -9,6 +9,10 @@
         (default 'Spotify not running')
     format_stopped: define output if spotify is not playing
         (default 'Spotify stopped')
+    sanitize_titles: whether to remove meta data from album/track title
+        (default True)
+    sanitize_words: which meta data to remove separated by |
+        (default 'stereo|mono|remaster|edit|bonus|extended|demo|explicit|version|feat')
```

Comment:
I'm trying to do that but I encounter a problem with default value in docstring
```
    sanitize_words: which meta data to remove
        (default ['stereo', 'mono', 'remaster', 'edit', 'bonus', 'extended', 'demo', 'explicit', 'version', 'feat'])
```
Results in 

> E501 line too long (116 > 99 characters)

and splitting it like that
```
        (default ['stereo', 'mono', 'remaster', 'edit',
                  'bonus', 'extended', 'demo', 'explicit',
                  'version', 'feat'])
```

Results in 
>`sanitize_words` default not in docstring

Can I somehow work this around without adding it to IGNORE_ITEM?